### PR TITLE
Resolves OKTA-328487 - adding custom attributes

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?**
Added coverage to the SCIM reference guides that explain what happens when a user is created in a downstream app that has custom user attributes defined in the schema.
- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-328487](https://oktainc.atlassian.net/browse/OKTA-328487)

### CODE REVIEW

**Review by connorrowe-okta — APPROVED** [review]
🚢

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 1]
```diff
@@ -183,6 +183,69 @@ Content-Type: application/json
 }
 ```
 
+However, if your SCIM server has one or more custom attributes that you want added to a new user's application profile, then any custom attributes that are defined in your application schema for user profiles are applied to that user’s application profile during the user creation operation. The request to the SCIM server looks like the following:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:186]: this sentence is uber long and has past tense. Try to keep to 25 words each.

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 2]
```diff
@@ -183,6 +183,69 @@ Content-Type: application/json
...
+    "active": true,
+    "userType": "Contractor"
+    "groups": [],
+    "meta": {
+        "resourceType": "User"
+    }
+}
+```
+
+In this example, the `externalID` and `userType` attributes were not included in the original POST method request, but are generated and returned in the SCIM server response.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:245]: were not -> weren't

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md [hunk 3]
```diff
@@ -183,6 +183,69 @@ Content-Type: application/json
...
+
+>**Note:** If you have your custom attribute defined in your Okta application integration settings as an App to Okta mapping, the custom attributes are not applied to the Okta user profile until either the user profile is imported from the SCIM application or the administrator performs a Force Sync operation.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:247]: are not -> aren't

Another uber long sentence. Try to keep to 25 words each.

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 4]
```diff
@@ -155,6 +155,69 @@ Content-Type: text/json;charset=UTF-8
...
+    "active": true,
+    "userType": "Contractor"
+    "groups": [],
+    "meta": {
+        "resourceType": "User"
+    }
+}
+```
+
+In this example, the `externalID` and `userType` attributes were not included in the original POST method request, but are generated and returned in the SCIM server response.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:217]: were not -> weren't

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md [hunk 5]
```diff
@@ -155,6 +155,69 @@ Content-Type: text/json;charset=UTF-8
...
+
+>**Note:** If you have your custom attribute defined in your Okta application integration settings as an App to Okta mapping, the custom attributes are not applied to the Okta user profile until either the user profile is imported from the SCIM application or the administrator performs a Force Sync operation.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:219]: are not -> aren't

Another uber long sentence. Try to keep to 25 words each.

**Review by susanharper-okta — APPROVED** [review]
couple of items. approving so that I don't hold you up.

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md (hunk 1 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:186]: Changed to :
There is another scenario if your SCIM server has custom attributes that you want to add for any new user. Any custom attributes defined in your application schema for user profiles are applied to the user's application profile when the user is created.

File: packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md (hunk 3 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md:247]: The best I can trim it to is:

> If your custom attributes are defined in your Okta integration (as an App to Okta mapping), the custom attributes aren't applied to the Okta user profile until an admin runs an import from the SCIM application or a Force Sync operation.

File: packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md (hunk 5 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md:219]: The best I can trim it to is:

> If your custom attributes are defined in your Okta integration (as an App to Okta mapping), the custom attributes aren't applied to the Okta user profile until an admin runs an import from the SCIM application or a Force Sync operation.

**Pull request merged**